### PR TITLE
add ubuntu 20.04 for owncloud-complete-10.5.0beta1

### PR DESCRIPTION
**Pull request by jnweiger**

### CODE REVIEW

File: v20.04/manifest.tmpl [hunk 1]
```diff
@@ -0,0 +1,18 @@
+image: owncloud/ubuntu:19.10
```
Comment (davitol) [inline v20.04/manifest.tmpl:1]: @jnweiger why 19.10 version for these images? Shoulnd't it be 20.04

**Comment by jnweiger** [comment]
Docker arm32v7/ubuntu:20.04 upstream is broken. We don't need  to support 32bit platforms, do we? I'll skip that one combination from the matrix.

**Comment by xoxys** [comment]
Dropping arm32 will afaik affect users with e.g. raspberry pi

File: latest/manifest.tmpl [hunk 2]
```diff
@@ -1,16 +1,13 @@
-image: owncloud/ubuntu:latest
+image: owncloud/ubuntu:20.04
+tags:
+  - eoan
```
Comment (davitol) [inline latest/manifest.tmpl:3]: Is tag eoan right? Shouldn't be focal?

**Comment by DeepDiver1975** [comment]
> Dropping arm32 will afaik affect users with e.g. raspberry pi

this seems to be a temporal fix - we can bring this back in a future pr once upstream fixed the bug. and yes - raspberry users might still want to use it.

File: latest/manifest.tmpl (hunk 2 above)
Comment (DeepDiver1975) [reply latest/manifest.tmpl:3]: I cannot judge on this - who knows what this is about? @xoxys maybe? THX

**Comment by jnweiger** [comment]
> > Dropping arm32 will afaik affect users with e.g. raspberry pi
> this seems to be a temporal fix - we can bring this back in a future pr once upstream fixed the bug. and yes - raspberry users might still want to use it.

Yes, agreed. Let's call it a temporary fix. :-)

File: latest/manifest.tmpl (hunk 2 above)
Comment (xoxys) [reply latest/manifest.tmpl:3]: Right. 
- Ubuntu 19.10 = Eoan Ermine = tag "eoan"
- Ubuntu 20.04 LTS = Focal Fossa = tag "focal"

But I also dont know why `image: owncloud/ubuntu:latest` was changed... @jnweiger?

Comment (DeepDiver1975) [reply latest/manifest.tmpl:3]: latest is anyhow a bit strage - shouldn't latest be a simple link to what ever we define to be latest? Having an extra build on it's own code base fells wrong ....

Comment (jnweiger) [reply latest/manifest.tmpl:3]: oh, yes eoan is an ubuntu code name. I'll change that manifest to say focal

Not sure what latest was. It was 18.04 for long time and nobody cared to update to 19.10, so maybe remove latest completely? 
If it is possible with a link or tag, then fine with me. I have no idea, why that was a separate build.

**Comment by jnweiger** [comment]
fixed the two manifests in latest and v20.04 to be correct now. Thanks @davitol

**Pull request merged**